Utilities/DataFlow/src/TimeframeValidatorDevice.cxx

```diff
@@ -88,14 +93,17 @@ void o2::DataFlow::TimeframeValidatorDevice::Run()
 
     // Data header it at position - 1
     auto tpcHeader = reinterpret_cast<DataHeader *>(timeframeParts.At(tpcIndex)->GetData());
-    if (tpcHeader->dataDescription != "TPCCLUSTER")
+    if ((tpcHeader->dataDescription != "CLUSTERS") ||
+        (tpcHeader->dataOrigin != "TPC"))
```

Comment:
I'm puzzled here. It should be
```c++
    if ((tpcHeader->dataDescription != o2::Header::DataDescription("CLUSTERS")) ||
        (tpcHeader->dataOrigin != o2::Header::DataOrigin("TPC")))
```
or the corresponding definitions. But how does it compile at all when using the const char strings? Neither DataDescription nor DataOrigin (aka Descriptor<4>) does define an operator==(const char*)

Looks like such operators need to be forbidden explicitly, I did a quick test.